### PR TITLE
Add test script to travis config

### PR DESCRIPTION
**Pull request by hamishtaplin**
## Changes

- Added `yarn test` command to Travis config
- Runs tests before build to fail early

## How to test

- Check that current test/s are being run on Travis
- Write a test that fails

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ node_js:
 
 script:
   - yarn build
+  - yarn test
```
Comment (samiwel) [inline .travis.yml]: By default won't this enter watch mode?

Comment (samiwel) [inline .travis.yml]: I stand corrected. Build passed!

Comment (hamishtaplin) [inline .travis.yml]: @samiwel I think the `CI` env var must be a default on travis? If it's `true` it enter single run mode.

**Comment by hamishtaplin** [comment]
[Testing framework](https://trello.com/c/qppjPNPY/12-testing-framework)

**Pull request merged**